crates/bevy_text/src/text.rs

```diff
@@ -160,6 +160,7 @@ impl From<TextAlignment> for glyph_brush_layout::HorizontalAlign {
 #[derive(Clone, Debug, Reflect, FromReflect)]
 pub struct TextStyle {
     pub font: Handle<Font>,
+    // The size of the font in pixels. Note that this value may surprise users who have a custom scale factor set.
```

Comment:
you should describe how it interacts with scale factor: is it multiplied by it or does it ignore it?